src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let result = [];
```

Comment:
you don't change this variable
```suggestion
  const result = [];
```